install/kubernetes/cilium/charts/hubble-ui/templates/svc.yaml

```diff
@@ -1,13 +1,13 @@
 kind: Service
 apiVersion: v1
 metadata:
-  namespace: {{ .Release.Namespace }}
+  namespace: {{.Release.Namespace}}
   name: hubble-ui
 spec:
   selector:
-    k8s-app: hubble-ui
+    app: hubble-ui
   ports:
     - name: http
       port: 80
-      targetPort: 12000
+      targetPort: 8081
```

Comment:
we need to update few docs for this port changes as well ? For example, https://github.com/cilium/cilium/blob/master/Documentation/gettingstarted/hubble-enable.rst

Reply:
no, port-forward command will continue to work